tests/test_telemetry_plugin.py

```
@@ -297,7 +297,9 @@ def test_matches_with_valid_command(self, mock_bot_command):
 
         Verifies that the method checks all available commands and returns True upon finding a match.
         """
-        mock_bot_command.side_effect = lambda cmd, event: cmd == "batteryLevel"
+        mock_bot_command.side_effect = (
+            lambda cmd, event, require_mention=False: cmd == "batteryLevel"
```

Ruff (0.14.6)

Unused lambda argument: `event`

(ARG005)